pkg/apis/aws/validation/worker_test.go

```diff
@@ -75,5 +87,14 @@ var _ = Describe("ValidateWorkerConfig", func() {
 				"Field": Equal("volume.iops"),
 			}))))
 		})
+
+		It("should return an error if a non-supported volume type is specified", func() {
```

Comment:
```suggestion
		It("should return an error if iops is set for a non-supported volume type", func() {
```

Reply:
Actually, do we have a check, that only supported volume types are specified in `workers[].volume.type`?

Reply:
This check is done by Gardener's `ShootValidator` admission plugin.

Reply:
Sure, forgot that this is cross-checked with the CloudProfile...